### ISSUE TITLE
新功能 新建议

### ISSUE THREAD

**Issue by YatFanLan** [IviriusCommunity/Rebound#90]
您好，您的软件我分外喜欢，我了解到之后便下载并安装了[Rebound](https://github.com/IviriusCommunity/Rebound)软件并体验，winUI3和Windows11 当前正在使用的统一，但是 问题在于该软件并没有加入简体中文 导致使用简体中文的用户使用起来颇为不便，希望作者能够加入中文简体

**Comment by Lamparter** [comment]
I agree. @IviriusMain we should implement locale translation function; most of the strings are hardcoded aren't they?

**Comment by IviriusMain** [comment]
Yes
